client/src/constants.js

```diff
@@ -1 +1,3 @@
-export const POSTS_PER_PAGE = 5;
+const POSTS_PER_PAGE = 5;
+const REFRESH_TOKEN_TIME = 5000;
```

Comment:
I have just paid attention to the actual time period. I supposed that the access token expires quickly... But not so fast 😄. I saw somewhere that it can have 10 or 15 minutes to expire.
You might think while choosing the appropriate values in this way: the old server might respond to you for up to 30 seconds. So during this time, you should refresh your token 6 times until you get your first answer.
Or another example: calculate the performance for 1000 simultaneously active users and compare it to the actual safety need of your application.